packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/events/map_event.dart

```diff
@@ -152,6 +152,14 @@ class CircleTapEvent extends MapEvent<CircleId> {
   CircleTapEvent(super.mapId, super.circleId);
 }
 
+/// An event fired when a [GroundOverlay] is tapped.
+class GroundOverlayTapEvent extends MapEvent<GroundOverlayId> {
+  /// Build an GroundOverlayTap Event triggered from the map represented by `mapId`.
```

Comment:
Nit: "a", not "an". (I'm not sure why a bunch of these methods have this mistake, but let's not continue replicating it.)